### PR TITLE
Add Scaled Exponential Linear Unit (SELU) activation 

### PR DESCRIPTION
**Pull request by vipulraheja**
[Self-Normalizing Neural Networks](https://arxiv.org/abs/1706.02515)
- [x] A new activation function called SELU
- [ ] A new Dropout function called AlphaDropout (set to a default value around 0.1 for lecun_normal)
- [ ] weight initialization technique (lecun_normal).
- [ ] Unit tests

### CODE REVIEW

**Comment by rbharath** [comment]
@vipulraheja Thanks for the contribution! Much appreciated :)

It looks you're currently failing the yapf test. Make sure to run yapf to format your code (see README for instructions). Should be good to merge after.

File: deepchem/nn/model_ops.py [hunk 1]
```diff
@@ -645,6 +645,12 @@ def relu(x, alpha=0., max_value=None):
   return x
 
 
+def selu(x):
```
Comment (lilleswing) [inline deepchem/nn/model_ops.py]: Can you add a link to the https://arxiv.org/pdf/1706.02515.pdf as a docstring?  These number are :ghost:

Comment (vipulraheja) [inline deepchem/nn/model_ops.py]: @lilleswing For sure! Thanks for pointing it out.

**Comment by rbharath** [comment]
@vipulraheja Looks great! Going to go ahead and merge. Would welcome future PRs for alpha-dropout more unit tests :)

LGTM.

**Comment by vipulraheja** [comment]
@rbharath Oh thanks! I was actually working on the tests but ran into some setup issues. Will create the respective PRs as you advised :)

**Pull request merged**